corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -0,0 +1,49 @@
+{% extends "style/bootstrap3/base_section.html" %}
```

Comment:
Just checking, but did you consider making app_diff.html and download_index.html derive from a new base template? There's a non-trivial amount of shared markup/js.